installer/pkg/workflow/install.go

```diff
@@ -14,27 +14,19 @@ func InstallWorkflow(clusterDir string) Workflow {
 	return Workflow{
 		metadata: metadata{clusterDir: clusterDir},
 		steps: []step{
-			refreshConfigStep,
-			generateClusterConfigMaps,
 			readClusterConfigStep,
+			generateTerraformVariablesStep,
 			generateTLSConfigStep,
 			generateClusterConfigMaps,
 			installAssetsStep,
 			generateIgnConfigStep,
 			installTopologyStep,
-			installBootstrapStep,
 			installMachinesStep,
+			installBootstrapStep,
```

Comment:
nit: I don't think you need to move `installBootstrapStep` after `installMachineStep` here.  You just clobber it in 532e4a3a3 anyway when you consolidate `installMachineStep` into `installInfraStep`, and the bootstrap and master/worker creation should be able to happen simultaneously anyway.